rust-bins/src/bin/keygen.rs

```diff
@@ -319,4 +712,136 @@ mod tests {
             hex::decode("171cb88b1b3c1db25add599712e36245d75bc65a1a5c9e18d76f9f2b1eab4012")
         );
     }
+
+    /// Test correct generation of BIP39 sentences.
+    /// Values are taken from https://github.com/trezor/python-mnemonic/blob/master/vectors.json
+    #[test]
+    pub fn test_bip39_generation() {
+        let bip39_vec: Vec<_> = include_str!("data/BIP39English.txt")
```

Comment:
Use the constant I mentioned in preceding comments to avoid including the same file 3 times.